zgw/besluiten-api/src/main/kotlin/com/ritense/besluitenapi/BesluitenApiPlugin.kt

```diff
@@ -99,28 +104,25 @@ class BesluitenApiPlugin(
         @PluginActionProperty uiterlijkeReactieDatum: LocalDate?,
         @PluginActionProperty createdBesluitUrl: String?,
     ) {
-
         val documentId = UUID.fromString(execution.businessKey)
         val zaakUrl = zaakUrlProvider.getZaakUrl(documentId)
-
-        logger.debug { "Creating besluit for zaak $zaakUrl of type $besluittypeUrl" }
-
-        val besluit = createBesluit(
-            zaakUrl = zaakUrl,
-            besluittypeUrl = URI(besluittypeUrl),
-            ingangsdatum = ingangsdatum ?: LocalDate.now(),
-            toelichting = toelichting,
-            bestuursorgaan = bestuursorgaan,
-            vervaldatum = vervaldatum,
-            vervalreden = vervalreden,
-            publicatiedatum = publicatiedatum,
-            verzenddatum = verzenddatum,
-            uiterlijkeReactieDatum = uiterlijkeReactieDatum
-        )
-
-        createdBesluitUrl?.let {
-            logger.debug { "Settings resulting variable $it to ${besluit.url}" }
-            execution.setVariable(it, besluit.url)
+        withLoggingContext("documentId" to documentId.toString()) {
```

Comment:
Since this is the identifier of a valtimo entity, it should reference the document class. So either Document::class if it is on the classpath or the classname as a string if not.